src/main/Duke/virtue/VirtueTaskList.java

```diff
@@ -0,0 +1,168 @@
+package virtue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VirtueTaskList {
+    // The task list.
```

Comment:
nit: this comment may either be removed for now, or you can make it more descriptive as needed in the future.